fixtures/pages-functions-app/functions/r2/create.ts

```diff
@@ -1,8 +1,8 @@
 type Env = {
-	BUCKET: R2Bucket;
+	bucket: R2Bucket;
```

Comment:
Seems like an unnecessary change, or am I missing something?

Reply:
The name should be lowercase

Reply:
But this is not the bucket name, it is the binding name, which doesn't have the same validation requirements.